docs/current_docs/cookbook/snippets/continue-after-errors/python/main.py

```diff
@@ -27,9 +27,9 @@ async def test(self) -> TestResult:
                 dag.container()
                 .from_("alpine")
                 # add script with execution permission to simulate a testing tool.
-                .with_new_file("run-tests", SCRIPT, permissions=0o750)
-                # if the exit code isn't needed: "run-tests; true"
-                .with_exec(["sh", "-c", "/run-tests; echo -n $? > /exit_code"])
+                .with_new_file("/run-tests", SCRIPT, permissions=0o750)
+                # run-tests but allow any return code
+                .with_exec(["/run-tests"], expect=ReturnType.ANY)
```

Comment:
For consistency (remove the import too):
```suggestion
                .with_exec(["/run-tests"], expect=dagger.ReturnType.ANY)
```